src/dune_rules/gen_rules.ml

```diff
@@ -136,7 +136,7 @@ end = struct
     | Cinaps.T cinaps ->
       let+ () = Cinaps.gen_rules sctx cinaps ~dir ~scope in
       empty_none
-    | Mdx.T mdx ->
+    | Mdx.T mdx when Expander.eval_blang expander (Mdx.enabled_if mdx) ->
```

Comment:
I was looking how we handle `enabled_if` for `(test ...)` and it's a bit different: we push the enabled_if deeper. Maybe we should do the same here. /cc @rgrinberg

Reply:
Do you mean passing a `~enabled_if` boolean in the mdx module?

Reply:
Welcome back @emillon :)

I can't quite remember. Looking again, it only matters if we the stanza allows the user to write a custom alias, as in this case we need to still make sure that Dune knows the alias is valid.

Since the mdx stanza doesn't allow that, the current code seems fine.